Automattic/wp-calypso
# Moves existing devDependencies to dependencies

#### Changes proposed in this Pull Request

* Moves `devDependencies` into `dependencies`. This is a pre-requisite to moving to `yarn`.

#### Testing instructions

Checkout the branch and do a diff of `package-lock.json` against master. Use `grep` to remove irrelevant bits (lots of changes in `dev: true` and extra commas for `integrity`). For example:

```
git diff -U0 master.. -- package-lock.json | grep -v '"dev": true' | grep -v '@@ '| grep  -v '"integrity"' | grep -v '"resolved"'
```

The result should show any effective change in the dependency tree. You'll see there are very few changes, all caused by applying different hoisting rules to `wp-prettier@1.19.1` and `is-number`
